projects/plugins/jetpack/modules/sso.php

```diff
@@ -79,6 +82,23 @@ public static function module_configure_button() {
 		Jetpack::enable_module_configurable( __FILE__ );
 	}
 
+	/**
+	 * Safety heads-up added to the logout messages when SSO is enabled.
+	 * Some folks on a shared computer don't know that they need to log out of WordPress.com as well.
+	 *
+	 * @param WP_Error $errors WP_Error object.
+	 */
+	public function sso_reminder_logout_wpcom( $errors ) {
+		if ( ( new Host() )->is_wpcom_platform() ) {
+			return $errors;
+		}
+
+		if ( ! empty( $errors->errors['loggedout'] ) ) {
+			$errors->add( 'jetpack-sso-show-logout', __( 'If you are on a shared computer, remember to also log out of wordpress.com.', 'jetpack' ), 'message' );
```

Comment:
This works well for me. Should we add a link to `https://wordpress.com/me`, where folks will find a logout button?

```suggestion
			$logout_message = wp_kses(
				sprintf(
					/* translators: %1$s is a link to the WordPress.com account settings page. */
					__( 'If you are on a shared computer, remember to also <a href="%1$s">log out of WordPress.com</a>.', 'jetpack' ),
					'https://wordpress.com/me'
				),
				array(
					'a' => array(
						'href' => array(),
					),
				)
			);
			$errors->add( 'jetpack-sso-show-logout', $logout_message, 'message' );
```